tests/e2escenarios/e2e_source_test.go

```diff
@@ -86,6 +86,9 @@ var _ = Describe("odo source e2e tests", func() {
 	Context("odo component creation", func() {
 
 		It("Should be able to deploy a python source application", func() {
+			if runtime.GOARCH == "ppc64le" {
```

Comment:
This is a pre test condition. Please add it in ```JustBeforeEach(func() {})``` of the same ```Context("", func() {})```

Reply:
@amitkrout There are 2 other tests in this context. Adding the condition in `JustBeforeEach(func() {})` of the same `Context("", func() {})` would skip the other 2 tests as well. Hence added this check only to this one test which is to be skipped.
I have followed existing similar checks as below and few more:-
https://github.com/openshift/odo/blob/master/tests/integration/plugin_handler_test.go#L46
https://github.com/openshift/odo/blob/master/tests/integration/devfile/cmd_devfile_push_test.go#L965
https://github.com/openshift/odo/blob/master/tests/integration/operatorhub/cmd_service_test.go#L416